src/current/_includes/releases/v24.3/v24.3.0-beta.2.md

```diff
@@ -0,0 +1,33 @@
+## v24.3.0-beta.2
+
+Release Date: October 28, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-beta-2-sql-language-changes">SQL language changes</h3>
+
+- If a table is the destination of a logical data replication stream, then only schema change statements that are deemed safe are allowed on the table. Safe statements are those that do not result in a rebuild of the primary [index]({% link v24.3/indexes.md %}) and do not create an index on a virtual [computed column]({% link v24.3/computed-columns.md %}). [#133266][#133266]
+
+<h3 id="v24-3-0-beta-2-operational-changes">Operational changes</h3>
+
+- Two new metrics are available `sql.crud_query.count` and `sql.crud_query.started.count` that measure the number of [`INSERT`]({% link v24.3/insert.md %})/[`UPDATE`]({% link v24.3/update.md %})/[`DELETE`]({% link v24.3/delete.md %})/[`SELECT`]({% link v24.3/selection-queries.md %}) queries executed and started respectively. [#133198][#133198]
+- When creating a logical data replication stream, any [user-defined types]({% link v24.3/create-type.md %}) in the source and destination are now checked for equivalency. This allows for creating a stream that handles user-defined types without needing to use the `WITH SKIP SCHEMA CHECK` option as long as the stream uses `mode = immediate`. [#133274][#133274]
```

Comment:
```suggestion
- When creating a logical data replication stream, any [user-defined types]({% link v24.3/create-type.md %}) in the source and destination are now checked for equivalency. This allows for creating a stream that handles user-defined types without needing to use the `WITH SKIP SCHEMA CHECK` clause as long as the stream uses `mode = immediate`. [#133274][#133274]
```

Reply:
It's an option. That't how we're describing those in the docs.

Reply:
Leaving as "option" per Kathryn's comment above.